appserver/payara-appserver-modules/microprofile/openapi/src/main/java/fish/payara/microprofile/openapi/impl/model/util/ModelUtils.java

```diff
@@ -395,7 +371,6 @@ public static boolean isAnnotationNull(Annotation annotation) {
         if (annotation == null) {
```

Comment:
Might it now be possible to move this static method into the new AnnotationInfo class?

Reply:
Possible yes, I'd think doing so conflates the classes purpose with that of a utility around annotations which it is not (yet) so I'd not do that unless you think this still is nicer than having it in `ModelUtils`. `AnnotationInfo` is used as one instance per type with state so a general annotation helper fits name wise but not so much context wise. WDYT?

Reply:
That's a fair comment, I'd just be keen to remove things from ModelUtils whenever I see a chance ;) The class exists just because I perceived it as necessary, but I don't really like it.

I agree that the scope of the class doesn't make much sense in this context though, so I'm happy to leave it as-is.